samples/matter/basic/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+
+project(matter-basic)
+
+set(COMMON_ROOT ${CMAKE_CURRENT_LIST_DIR}/../common)
+set(NLIO_ROOT ${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/third_party/nlio/repo)
+include(${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/app/enable-gnu-std.cmake)
+
+# NORDIC SDK APP START
+target_include_directories(app PRIVATE src ${COMMON_ROOT}/src ${NLIO_ROOT}/include)
+
+target_sources(app PRIVATE
+	src/app_task.cpp
```

Comment:
Do you use clang format? It looks like indent was not aligned properly.

Reply:
clang-format doesn't format CMake files

Reply:
What would be a proper way to format it?

Reply:
I think you need to format it by hand unfortunately taking existing samples as a reference. In this case probably indenting the source files to the level of the `app` target will be enough.